Sources/Sentry.swift

```diff
@@ -96,7 +99,7 @@ import Foundation
 	- Parameter level: The severity of the message
 	*/
 	@objc public func captureMessage(message: String, level: SentrySeverity = .Info) {
-		let event = Event(message, level: level)
+		let event = Event(message, level: level, release: releaseVersion)
```

Comment:
@fizker Instead of here, the release should probably added into the `captureEvent` function inside of this if statement - https://github.com/revisohq/sentry-swift/blob/3c56b271ac9e40dcda84a1ce95c51e16a50ce7a0/Sources/Sentry.swift#L119-L129